--- Makefile	2025-05-04 12:40:49.581255000 +0300
+++ Makefile_andrew	2025-05-04 19:16:42.320899054 +0300
@@ -45,6 +45,7 @@
   TITLE:=DNS and DHCP server
   URL:=http://www.thekelleys.org.uk/dnsmasq/
   USERID:=dnsmasq=453:dnsmasq=453
+  DEPENDS+=+libpcre
 endef

 define Package/dnsmasq
@@ -149,6 +150,7 @@
 		$(if $(CONFIG_PACKAGE_dnsmasq_$(BUILD_VARIANT)_broken_rtc),-DHAVE_BROKEN_RTC) \
 		$(if $(CONFIG_PACKAGE_dnsmasq_$(BUILD_VARIANT)_tftp),,-DNO_TFTP)
 	COPTS += $(if $(CONFIG_LIBNETTLE_MINI),-DNO_GMP,)
+	COPTS += -DHAVE_REGEX -DHAVE_REGEX_IPSET
 else
 	COPTS += -DNO_AUTH -DNO_IPSET -DNO_ID
 endif
