Index: dnsmasq/Makefile
===================================================================
--- dnsmasq.orig/Makefile	2021-12-27 16:50:02.807894928 +0300
+++ dnsmasq/Makefile	2021-12-27 21:08:18.006310064 +0300
@@ -43,6 +43,7 @@
   CATEGORY:=Base system
   TITLE:=DNS and DHCP server
   URL:=http://www.thekelleys.org.uk/dnsmasq/
+  DEPENDS+=+libpcre
 endef
 
 define Package/dnsmasq
@@ -128,7 +129,7 @@
 TARGET_CFLAGS += -flto
 TARGET_LDFLAGS += -flto=jobserver
 
-COPTS = -DHAVE_POLL_H \
+COPTS = -DHAVE_POLL_H -DHAVE_REGEX -DHAVE_REGEX_IPSET \
 	$(if $(CONFIG_IPV6),,-DNO_IPV6)
 
 ifeq ($(BUILD_VARIANT),nodhcpv6)
